src/DataStructures/Mesh.hpp

```diff
@@ -144,21 +145,48 @@ class Mesh {
   }
 
   /*!
-   * \brief The quadrature chosen in dimension `d` of the grid.
+   * \brief The quadrature chosen in dimension \p d of the grid (zero-indexed).
    */
   Spectral::Quadrature quadrature(const size_t d) const noexcept {
     return gsl::at(quadratures_, d);
   }
 
   /*!
-   * \brief Returns a mesh with dimension `d` removed.
+   * \brief Returns a Mesh with dimension \p d removed (zero-indexed).
```

Comment:
I have no idea what this `\p` does. @kidder @wthrowe did you look at this rendered?